yb-voyager/src/query/queryissue/issues_ddl.go

```diff
@@ -523,7 +523,7 @@ func NewPathDatatypeIssue(objectType string, objectName string, sqlStatement str
 var polygonDatatypeIssue = issue.Issue{
 	Type:        POLYGON_DATATYPE,
```

Comment:
as discussed lets rename this and other ones
`POLYGON_DATATYPE` --> `UNSUPPORTED_DATATYPE_LIVE_MIGRATION_POLYGON`